platform.md

```diff
@@ -179,3 +189,37 @@ group = [
 Where:
 
 - Both `id` and `version` MUST be present for each buildpack object in a group.
+
+### project.toml (TOML)
+
+```toml
+[source]
+type = "<source type>"
+
+[source.version]
+
+[source.metadata]
+
+```
+
+### io.buildpacks.project OCI Image label
```

Comment:
Can we express this in TOML (using the pattern already established), and then indicate that the label is in JSON?

Reply:
I disagree with this, the content of the OCI image config label is in json so it makes sense to define it with json schema in the specification.

Reply:
As mentioned below, we've now incorporated this change.